### PR TITLE
Tweaks to team building team-in-progress

### PR DESCRIPTION
**Pull request by buoyad**
- Focus search input after someone is added
- Make non-kb "usernames" all black, they were grey before
- Make scrollbar align with team-in-progress container
- Don't show service logos - show avatar placeholders for all services
- Re-add '@service' suffix to SBS people
- Show tooltip when hovering entire container, not just (x)
- Make (x) black on hover

Punted:
- Format phone numbers (Y2K-588)

cc @keybase/picnicsquad @keybase/y2ksquad

### CODE REVIEW

File: shared/common-adapters/name-with-icon/index.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ class NameWithIcon extends React.Component<NameWithIconProps> {
         usernames={[username]}
         colorBroken={this.props.colorBroken}
         colorFollowing={this.props.colorFollowing}
-        colorYou={this.props.notFollowingColorOverride}
+        colorYou={this.props.notFollowingColorOverride || true}
```
Comment (chrisnojima) [inline shared/common-adapters/name-with-icon/index.tsx]: ????

Comment (buoyad) [reply shared/common-adapters/name-with-icon/index.tsx]: `Usernames` expects a string color to override your own username or `true` to set it to black. The ticket requested we change it in team building but it seemed fine to change it for `NameWithIcon`

Comment (chrisnojima) [reply shared/common-adapters/name-with-icon/index.tsx]: this makes it always true?

Comment (buoyad) [reply shared/common-adapters/name-with-icon/index.tsx]: This makes it so your own username in `NameWithIcon` is black by default

Comment (chrisnojima) [reply shared/common-adapters/name-with-icon/index.tsx]: Why have the parameter then

**Pull request merged**